packages/jest-jasmine2/src/jasmineAsyncInstall.ts

```diff
@@ -187,6 +191,8 @@ function makeConcurrent(
 
     return spec;
   };
+  concurrentFn.each = () => {};
```

Comment:
In `jest-jasmine2` each is binded after the function is made concurrent, so I had it as a no-op as it will eventually be added. In the previous iterations I had to do a `ts-ignore`. I'm open to change it

Reply:
a code comment explaining why makes sense to me